src/Nethermind/Nethermind.TxPool/Collections/SortedPool.cs

```diff
@@ -18,102 +18,99 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Runtime.CompilerServices;
+using Nethermind.Core.Collections;
 
 namespace Nethermind.TxPool.Collections
 {
-    public class SortedPool<TKey, TValue>
+    public class SortedPool<TKey, TValue, TGroup>
```

Comment:
why changing existing class instead of adding a new one?

Reply:
This class has just one use currently - TxPool. I can bring back old implementation if needed. We currently don't need it though.